examples/mnist/lenet.jl

```diff
@@ -42,4 +42,7 @@ model = mx.FeedForward(lenet, context=mx.gpu())
 optimizer = mx.SGD(lr=0.05, momentum=0.9, weight_decay=0.00001)
 
 # fit parameters
-mx.fit(model, optimizer, train_provider, n_epoch=20, eval_data=eval_provider)
+mx.fit(model, optimizer, train_provider,
+       n_epoch=20,
+       eval_data=eval_provider
```

Comment:
This line needs a comma `,` at the end, yes?

Reply:
ha, 👍